src/python/ensembl/io/genomio/genome_metadata/prepare.py

```diff
@@ -122,9 +122,9 @@ def add_assembly_version(genome_data: Dict) -> None:
     assembly = genome_data["assembly"]
     if not "version" in assembly:
         accession = assembly["accession"]
-        values = accession.split(".")
-        if (len(values) == 2) and values[1]:
-            assembly["version"] = int(values[1])
+        version = accession.partition(".")[2]
+        if version:
+            assembly["version"] = int(version)
```

Comment:
`accession.partition(".")` works slight different, as it always returns a tuple with 3 values. But if `.` (or the version number afterwards) is not present, the last 2 values will be `""`. Thus, this check is to prevent a `ValueError` raised when attempting to convert `""` to integer.